adsh/src/components/NavProfile.js

```diff
@@ -1,14 +1,74 @@
-import React from 'react'
-import { styled } from 'styled-components';//import used to make inline css easier
+import React, {useState} from 'react'
+
+// import { styled } from 'styled-components';//import used to make inline css easier
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';//import used to add icons, need to install package
-import { faEye } from '@fortawesome/free-solid-svg-icons';//import used to add the eye icon, need to install package
+import { faUser } from '@fortawesome/free-solid-svg-icons';//import used to add the eye icon, need to install package
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faRobot} from '@fortawesome/free-solid-svg-icons';
+import '../styles/NavProfile.css';
 
 const NavProfile = () => {
     return (
         <div>
+        <NavBar>
+            <NavItems id='logo' icon= {faRobot} width="20px" height="20px">
+            </NavItems>
+            <div className='right-nav-items'>
+            <NavItems id='right-logo' icon= {faPlus} width="20px" height="20px">
+                <DropDown>
+                    <DropDownItem link="#">Post/Submit</DropDownItem>
+                </DropDown>
+            </NavItems>
+            <NavItems id='right-logo' icon= {faUser} width="20px" height="20px">
+                <DropDown>
+                    <DropDownItem link="#"> My Profile</DropDownItem>
+                    <DropDownItem link="#">Setting</DropDownItem>
+                    <DropDownItem link="#">Log Out</DropDownItem>
+                </DropDown>
+            </NavItems>
+            </div>
+        </NavBar>
+        </div>
+    );
+} 
+
+const NavBar = (props) => {
```

Comment:
What style are we using for functions with single varibles. This could just be const NavBar = props => {...}

Reply:
Lets be consistent with the style